File: lib/data/episode.dart
class Episode {
  int? episodeId;
  String? name;
  String? link;
  int? movieId;

  Episode({this.episodeId, this.name, this.link, this.movieId});

  Episode.fromJson(Map<String, dynamic> json) {
    episodeId = json['episodeId'];
    name = json['name'];
    link = json['link'];
    movieId = json['movieId'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['episodeId'] = episodeId;
    data['name'] = name;
    data['link'] = link;
    data['movieId'] = movieId;
    return data;
  }
}

File: netlexapiwebadmin/netlexapiwebadmin/Models/Episode.cs
using System;
using System.Collections.Generic;

namespace netlexapiwebadmin.Models
{
    public partial class Episode
    {
        public int EpisodeId { get; set; }
        public string? Name { get; set; }
        public string? Link { get; set; }
        public int? MovieId { get; set; }

        public virtual Movie? Movie { get; set; }
    }
}
